File: playground/php/src/agent-studio.php
<?php

$env = require_once '../loadEnv.php';

use Algolia\AlgoliaSearch\Api\AgentStudioClient;

$client = AgentStudioClient::create(
    $env['ALGOLIA_APPLICATION_ID'],
    $env['ALGOLIA_ADMIN_KEY']
);

$response = $client->listAgents();

echo 'List of agents:' . PHP_EOL;
foreach ($response['agents'] as $agent) {
    echo '- ' . $agent['name'] . ' (ID: ' . $agent['id'] . ')' . PHP_EOL;
}

File: playground/ruby/agent-studio.rb
require 'dotenv'
require 'algolia'

Dotenv.load('../.env')

client = Algolia::AgentStudioClient.create(ENV['ALGOLIA_APPLICATION_ID'], ENV['ALGOLIA_ADMIN_KEY'], 'us')

response = client.list_agents
puts 'List of agents:'
response.agents.each do |agent|
  puts "- #{agent.name} (ID: #{agent.id})"
end
